backend/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
```

Comment:
Nit: Neither the cors nor the plaid dependencies are used in this PR. In the future, only import a new dependency if they are actually going to be used.